guides/common/modules/proc_changing-the-repository-sets-status-for-a-host-in-project.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Changing_the_Repository_Sets_Status_for-a-Host-in_{project-context}_{context}"]
+= Changing the Repository Sets Status for a Host in {Project}
+
+Repository sets show repositories available to each host.
+A host will be able to access content from a given repository if that repository is enabled.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
+. Select the *Content* tab, then select the *Repository sets* subtab.
+On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
+. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
+. You can also bulk select the checkboxes on each table row, and use the vertical ellipsis icon at the top.
+. On the *Overview* tab, click *Content view details* to view the environment for the host.
```

Comment:
This is related to the note below, and shouldn't be a part of the procedure. It makes more sense as the last sentence of the note.

Reply:
Which part is related to the note below? Are you referring to On the *Overview* tab, click *Content view details* to view the environment for the host?

Reply:
Yeah, that sentence.